### PR TITLE
[444.hu] Website is no longer mixedcontent

### PR DESCRIPTION
**Pull request by TK-999**
http://444.hu/

### CODE REVIEW

File: src/chrome/content/rules/444.hu.xml [hunk 1]
```diff
@@ -1,15 +1,6 @@
-<!--
-	Nonfunctional domains:
-	- 4cdn.hu (image assets)
-	- img.444.hu (refused)
-	- q.444.hu (expired)
-	- cimlap.444.hu (expired)
```
Comment (J0WI) [inline src/chrome/content/rules/444.hu.xml]: Why did you removed those hosts? They still exist.

Comment (TK-999) [reply src/chrome/content/rules/444.hu.xml]: HTTPS is now correctly configured for these domains as well.

Comment (J0WI) [reply src/chrome/content/rules/444.hu.xml]: So they should be added as targets.

**Comment by jeremyn** [comment]
@TK-999 Are you still interested in working on this pull request?

**Comment by jeremyn** [comment]
I'm closing this pull request due to lack of response from @TK-999 . I've also made a new pull request https://github.com/EFForg/https-everywhere/pull/14125 that updates `444.hu.xml`.

**Pull request closed without merging**